src/shop/_components/welcomeGiftsBanner/WelcomeGiftsBanner.tsx

```diff
@@ -1,21 +1,27 @@
 import React from 'react'
 import { useTranslation } from 'react-i18next'
+import { useNavigation } from '@react-navigation/native'
+import { isBefore, isValid, parseISO, subMonths } from 'date-fns'
 
 import { GiftHalf } from '../../../_assets/images'
 import { Icon } from '../../../_components'
+import { TMainNavigationProp } from '../../../_routing'
 import { theme } from '../../../_styles/theme'
 import { useGetMe } from '../../../profile/_queries/useGetMe'
 import * as Styled from './style';
 
+const today = new Date();
```

Comment:
snap het wel, maar met nen date kan da in serieuze edge cases wel rare bugs opleveren

Reply:
Hoezo?

Reply:
Fixed ;-)